preLog.txt

```diff
@@ -1 +1 @@
-
+1.
```

Comment:
Poprosím změnit na hash commitu, měli jsme chybu v instrukcích. Důležité je si do své větve vložit aktuální hlavní větev (tedy main), případně můžeš projít instrukce komplet celé znovu.